tests/test_generate_files.py

```diff
@@ -334,3 +334,21 @@ def test_raise_undefined_variable_project_dir(tmpdir):
     assert error.context == {}
 
     assert not output_dir.join('testproject').exists()
+
+
+def test_external_templates(tmpdir):
+    output_dir = generate.generate_files(
```

Comment:
`output_dir` is a little confusing in this context, could we rename this to `project_dir` instead?